zcash_client_backend/src/decrypt.rs

```diff
@@ -33,7 +45,7 @@ pub struct DecryptedOutput {
     /// this is a logical output of the transaction.
     ///
     /// [`OutgoingViewingKey`]: zcash_primitives::keys::OutgoingViewingKey
-    pub outgoing: bool,
+    pub transfer_type: TransferType,
```

Comment:
- This is a change to a public type. Document it in the changelog.
- Per my above comment, a single instance of `TransferType` is insufficient to fully describe the possible transaction types. This needs to either be a set or a bitflag, or `TransferType` needs additional cases.